src/AppBundle/Controller/PageController.php

```diff
@@ -148,4 +148,13 @@ public function botTestingAction()
 
         return $this->render('page/bot-testing.html.twig');
     }
+
+    /**
+     * @Route("/adherer", name="page_adherer")
```

Comment:
Duplicate of `https://github.com/EnMarche/en-marche.fr/blob/master/src/AppBundle/Controller/MembershipController.php#L28`?